server/src/main/java/org/elasticsearch/index/mapper/MapperService.java

```diff
@@ -506,7 +506,7 @@ public boolean isMetadataField(String field) {
     }
 
     public boolean isMultiField(String field) {
-        return mappingLookup().isMultiField(field);
+        return mappingLookup().isShadowed(field) == false && mappingLookup().isMultiField(field);
```

Comment:
mmm I did not mean for you to rely on a method that I am planning on removing :) could we instead have all the logic within `MappingLookup#isMultiField`? Would `if (fieldMappers.containsKey(field) == false && fieldTypeLookup.getFieldType(field) == null)` work?

Reply:
We can test that values returned from `indexTimeLookup` and `fieldTypeLookup` are not the same, as runtime fields are not included in `indexTimeLookup`